examples/own_dataset/README.md

```diff
@@ -0,0 +1,13 @@
+# Example of using your own dataset
+## Usage
+```
+python train.py dataset.csv --label value1 value2
+```
```

Comment:
Please explain (a part of ) options of the script. At least `--label` needs description. But I think the option is the only one treat specially and it would be enough to write as "type `python train.py --help` to see complete options".